packages/oas-to-snippet/README.md

```diff
@@ -24,41 +24,62 @@ import petstore from './petstore.json';
 const formData = {
   query: { sort: 'desc' },
 };
 
-// This is a keyed object containing authentication credentials for the operation. The keys for
-// this object should match up with the `securityScheme` on the operation you're accessing, and
-// its value should either be a String, or an Object containing `user` and/or `pass` (for Basic
-// auth schemes).
+// This is a keyed object containing authentication credentials for the
+// operation. The keys for this object should match up with the `securityScheme`
+// on the operation you're accessing, and its value should either be a String,
+// or an Object containing `user` and/or `pass` (for Basic auth schemes).
 const auth = {
   oauth2: 'bearerToken',
 };
 
-// This is the language to generate a snippet to. See below for supported languages.
+// This is the language to generate a snippet to. See below for supported
+// languages.
 //
-// For supplying an alternative language target (like `axios` for `node`), you can do so by
-// changing this variable to an array: `['node', 'axios']`. For the full list of alternative
-// language targets that we support, see below.
+// For supplying an alternative language target (like `axios` for `node`), you
+// can do so by changing this variable to an array: `['node', 'axios']`. For the
+// full list of alternative language targets that we support, see below.
 const language = 'node';
 
-// This `registryIdentifier` option is only necessary when using the `['node', 'api']` language
-// combination. It controls how these snippets are used according to the code generation tooling
-// that we offer with https://api.readme.dev/.
-const registryIdentifier: '@petstore/v2.0#17273l2glm9fq4l5';
+// This will return an object containing `code` and `highlightMode`. `code` is
+// the generated code snippet, while `highlightMode` is the language mode you
+// can use to render it for syntax highlighting (with `codemirror` for example).
+const { code, highlightMode } = oasToSnippet(apiDefinition, operation, formData, auth, language });
+```
+
+### Plugins
+
+This library also supports the plugin system that we've built into [HTTPSnippet](https://npm.im/@readme/httpsnippet). We have a plugin for generating snippets for [ReadMe's API SDK generator](https://api.readme.dev) and this is how you would integrate and generate snippets for it:
 
-// This will return an object containing `code` and `highlightMode`. `code` is the generated code
-// snippet, while `highlightMode` is the language mode you can use to render it for syntax
-// highlighting (with @readme/syntax-highlighter, for example).
-const { code, highlightMode } = oasToSnippet(apiDefinition, operation, formData, auth, language, { openapi: { registryIdentifier } });
+```js
+import oasToSnippet from '@readme/oas-to-snippet';
+import httpsnippetClientAPIPlugin from 'httpsnippet-client-api';
+
+const snippet = await oasToSnippet(
+  petstore,
+  petstore.operation('/user/login', 'get'),
+  formData,
```

Comment:
wanna add some example usage/comments here that indicate how these variables are constructed?

Reply:
These exist above in the readme.

Reply:
woof my bad 😵‍💫